spec/models/zip_code_spec.rb

```diff
@@ -3,5 +3,75 @@
 require 'rails_helper'
 
 RSpec.describe ZipCode, type: :model do
-  it { should validate_presence_of(:d_codigo) }
+  context 'validations tests' do
+    it { should validate_presence_of(:d_codigo) }
+  end
+
+  context 'scopes tests' do
```

Comment:
I would suggest to move this into the factory for the ZipCode, for you to easily create any record for this class

Reply:
Done!